src/huggingface_hub/hf_api.py

```diff
@@ -142,39 +143,33 @@ def __repr__(self):
         return f"{self.__class__.__name__}({', '.join(items)})"
 
 
-class ModelFile:
+class ModelFile(RepoFile):
     """
-    Data structure that represents a public file inside a model, accessible from huggingface.co
+    Data structure that represents a public file inside a repo, accessible from huggingface.co
```

Comment:
If you're updating the docstring from `model` to `repo`, you can remove this docstring altogether as it'll re-use the inherited `RepoFile`. You could also revert the `model` -> `repo` change to have a more explicit docstring.

Reply:
(especially since `dataset` and `Space` are kept in the respective `XXXFile` classes defined below)